### DISCUSSION TITLE
Control and enrichment on same flow cell

### DISCUSSION THREAD

**Discussion by jennieli421** [LooseLab/readfish#307] [General]
I wonder if there is the option to select some pores for control and others for enrichment on the same flowcell using readfish.

**Comment by mattloose** [comment]
Hi, 

Yes - this can definitely be done - please see the documentation for some examples of control regions in a toml file.

 https://looselab.github.io/readfish/toml.html#analysis-regions

**Comment by jennieli421** [comment]
Is there a place that explains how to do analysis post-sequencing? How to separate the output of control region from enrichment region?

**Reply by mattloose** [reply]
Hi. 

You can have a look in the documentation here - https://looselab.github.io/readfish/readfish.console.html#module-readfish.entry_points.stats

However, it is better to look at examples of analysis in the literature.

If you have a new question best to open it in a separate discussion.

**Comment by jennieli421** [comment]
My version of readfish in conda is 
```
readfish                  0.0.11a4                 pypi_0    pypi
```
I am slightly confused which version that is. Also can I still run this version of readfish (following the old instructions) now?

**Reply by mattloose** [reply]
You should update to the latest version of readfish. The version you are running is pretty out of date and is not immediately compatible with these documents. 

I'd recommend making a new installation following the instructions in the documentation and taking it from there.

**Reply by jennieli421** [reply]
I am about to perform an experiment and all the designs were based on the old version... If not updating is not going to cause any issue I would prefer to stick to it

**Reply by mattloose** [reply]
So the previous design should work with the new version with just minor changes to the toml file. 

I can appreciate your concern about moving over. Typically we suggest running a control using a bulk file before doing an actual run (as described in the previous documentation and in the new updated docs here).

If you have run that test and it worked then you should be good to go.
